src/stack.js

```diff
@@ -1,5 +1,23 @@
 class Stack {
-
+  constructor() {
+    this.size = 0;
+    this.stack = {};
```

Comment:
I love that you used an object as your storage instead of an array. Arrays are easy for me to conceptualize and work with. But this shows that you have a grasp of the types of data stores that JS has to offer.